app/views/school-administrator/teachers/SchoolAdminTeacherListView.vue

```diff
@@ -92,6 +92,9 @@
 
           for (const teacher of this.administratedTeachers) {
             const trialRequest = teacher._trialRequest || {}
+            if (!trialRequest.organization) {
```

Comment:
This is currently handled on line 45 in the template.  There is a `v-else` that should switch the h4 to say "Other" using our i18n tooling.

Looks like the check is not working though?  It uses a typeof check, maybe we change it to a simple `!groupName`?

Reply:
Ah, nice! Did not follow this all the way through well enough - will update, thank you!